app/controllers/api/verify/password_confirm_controller.rb

```diff
@@ -96,6 +96,26 @@ def save_in_person_enrollment(user, profile)
         enrollment = InPersonEnrollment.create!(
           profile: profile,
           user: user,
```

Comment:
could we also set `current_address_matches_id` here, based on the value of `same_address_as_id` in the session?

Reply:
Added in 4291c72ef